mm2src/mm2_core/src/mm_ctx.rs

```diff
@@ -304,7 +302,10 @@ impl MmCtx {
     ///
     /// No checks in this method, the paths should be checked in the `fn fix_directories` instead.
     #[cfg(not(target_arch = "wasm32"))]
-    pub fn dbdir(&self) -> PathBuf { path_to_dbdir(self.conf["dbdir"].as_str(), self.rmd160()) }
+    pub fn dbdir(&self, db_id: Option<&str>) -> PathBuf {
+        let db_id = db_id.map(|t| t.to_owned()).unwrap_or_else(|| self.rmd160.to_string());
```

Comment:
we have several places of code like this, to unwrap optional db_id and use default value if needed.
Maybe it is good to make a function like unwrap_db_id()

Reply:
Not insisting. 
As examples, we have fn unwrap_single_addr and fn unwrap_w for custom unwraps, providing more code reuse